thorlcr/slave/thslavemain.cpp

```diff
@@ -393,6 +393,7 @@ int main( int argc, const char *argv[]  )
 #else
         globals.setown(loadConfiguration(globals, argv, "thor", "THOR", nullptr, nullptr, nullptr, false));
 #endif
+        workunitGraphCacheEnabled = globals->getPropBool("expert/@workunitGraphCacheEnabled", workunitGraphCacheEnabled);
```

Comment:
I think this is too early.  (See the comment in the code below....)
I would expect a similar call in the thor master. 

@jakesmith Does the slave load the graph from the workunit?  Does it reload it for each subgraph?

Reply:
Not per subgraph, but per Thor 'job' (i.e. each workflow graph)

Reply:
>I think this is too early. (See the comment in the code below....)

yes, would be good to move within RegisterSelf next to similar expert tests,
and so that's bare-metal compatible, should use getExpertOptBool, i.e.:

Reply:
>I would expect a similar call in the thor master.

in master, a getExpertOptBool call should go in CJobManager::execute

Reply:
Fixed